<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1478</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget
{

border-image: url(:/anh/neneee.png);
}
#frame
{
background-color: rgba(0, 0, 0, 0.3);

}
#widget_4
{
border-image: url(:/anh/tay.png);
 border-top-left-radius: 50px;

}
#widget_5
{

    background-color: #fefeff;
    border-bottom-right-radius: 50px;
}
QPushButton#B1 {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(11, 131, 120, 219), 
                                      stop:1 rgba(85, 98, 112, 226));
    color: rgba(255, 255, 255, 210);
    border-radius: 5px;
}

QPushButton#B1:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(150, 123, 111, 219), 
                                      stop:1 rgba(85, 81, 84, 226));
}

QPushButton#B1:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(150, 123, 111, 255);
}
QPushButton#B2 {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(11, 131, 120, 219), 
                                      stop:1 rgba(85, 98, 112, 226));
    color: rgba(255, 255, 255, 210);
    border-radius: 5px;
}

QPushButton#B2:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(150, 123, 111, 219), 
                                      stop:1 rgba(85, 81, 84, 226));
}

QPushButton#B2:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(150, 123, 111, 255);
}
QPushButton#B3 {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(11, 131, 120, 219), 
                                      stop:1 rgba(85, 98, 112, 226));
    color: rgba(255, 255, 255, 210);
    border-radius: 5px;
}

QPushButton#B3:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(150, 123, 111, 219), 
                                      stop:1 rgba(85, 81, 84, 226));
}

QPushButton#B3:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(150, 123, 111, 255);
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>100</number>
             </property>
             <property name="topMargin">
              <number>150</number>
             </property>
             <property name="rightMargin">
              <number>100</number>
             </property>
             <property name="bottomMargin">
              <number>200</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QWidget" name="widget_6" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>70</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="leftMargin">
                    <number>50</number>
                   </property>
                   <property name="topMargin">
                    <number>20</number>
                   </property>
                   <property name="rightMargin">
                    <number>20</number>
                   </property>
                   <item alignment="Qt::AlignHCenter">
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <pointsize>30</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Register</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_8" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>800</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>12</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="Name_r">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>300</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 240);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string>Full name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Username_r">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>300</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 240);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="placeholderText">
                      <string>User name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Password_r1">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>300</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 240);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="placeholderText">
                      <string>Password</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Password_r2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>300</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 240);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string>Confirm password</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_7" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>500</width>
                    <height>80</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>70</number>
                   </property>
                   <property name="topMargin">
                    <number>20</number>
                   </property>
                   <property name="rightMargin">
                    <number>50</number>
                   </property>
                   <property name="bottomMargin">
                    <number>20</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="B3">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>300</width>
                       <height>100</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton#pushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(11, 131, 120, 219), 
                                      stop:1 rgba(85, 98, 112, 226));
    color: rgba(255, 255, 255, 210);
    border-radius: 5px;
}

QPushButton#pushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, 
                                      stop:0 rgba(150, 123, 111, 219), 
                                      stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(150, 123, 111, 255);
}
</string>
                     </property>
                     <property name="text">
                      <string>Register</string>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>16</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
